_posts/2021-01-13-intro-python.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Introduction to Python"
+text: "A brief introduction to Python for beginners.
+"
```

Comment:
Has it always been adding this extra newline? Does that actually work?

Reply:
I've just worked off of previous formatting (which included this) but I could remove it and see what happens

Reply:
Wait, I didn't realize this wasn't `events.yml` - yeah, it seems the newline is being automatically added. Weird